conf.py

```diff
@@ -6,6 +6,8 @@ class Settings(BaseSettings):
     DYNAMODB_HOST: str = "127.0.0.1"
     DYNAMODB_PORT: int = 8000
     STORY_TABLE_NAME: str = "stories"
+    DYNAMODB_URL: str = f"http://{DYNAMODB_HOST}:{DYNAMODB_PORT}"
```

Comment:
If we read `DYNAMODB_HOST`/`DYNAMODB_PORT` from env, `DYNAMODB_URL` will be invalid.